docs/assets/diagrams/linking/2-1-pre-linking.puml

```diff
@@ -0,0 +1,100 @@
+@startuml
+
+title: Discovery 
+
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+' start flow
+loop Daily
+  Server -> Mojaloop++: ""**GET /participants**""
+  Server <-- Mojaloop: ""202 Accepted""
+
+  Mojaloop -> Mojaloop: Lookup data
+
+  rnote left of Mojaloop #Light
+  ""{ ""
+  ""  participants: [""
+  ""    {""
+  ""      "fspId": "fspa",""
+  ""      "name": "FSP A"""
+  ""    },""
+  ""    {""
+  ""      "fspId": "fspb",""
+  ""      "name": "FSP B"""
+  ""    }""
+  ""  ] ""
+  ""} ""
+  end rnote
+
+  Mojaloop -> Server ++: ""**PUT /participants**""
+  Mojaloop <-- Server: ""200 OK""
+  deactivate Mojaloop
+
+  rnote left of Server #Light
+    ""db = firebase.firestore()""
+    ""batch = db.batch()""
+    ""docRef = db.collections("participants")""
+    """"
+    ""db.collection().get().then(querySnapshot => {""
+    ""  querySnapshot.forEach(doc => {""
+    ""    batch.delete(doc.ref)""
+    ""  })""
+    ""})""
+    """"
+    ""req.payload.participants.forEach(entry => {""
```

Comment:
do we need this level of detail?

Reply:
after thinking more about this, I think not... removed :)